leaf-ui/index.html

```diff
@@ -166,14 +166,13 @@ <h3 style="text-align:left; color:#1E85F7; margin-bottom:5px;">Relative Intentio
 		      <button id="symbolic-btn" class="blue-btn btn">Model Constraints</button>-->
             <button id="analysis-btn" class="green-btn btn" style="border-radius:none;">Analysis</button>
         </div>
-        
-
+         
         <div id="stencil" class="left-panel"><label>Stencil</label></div>
         <div id="analysis-sidebar" class="left-panel"><h3 style="text-align:left; color:#181b1fe3; margin-bottom:5px; margin-left: 10px;">Analysis
             <div class="addConfig" style="display:inline">
                 <i class="fa fa-plus" id="addIntent" style="font-size:30px; float:right;  margin-bottom:5px; margin-right:20px;"></i>
             </div></h3>
-            <div id="configurations" class="left-panel" style="margin-top:20px"></div>
+            <div id="configurations" class="left-panel" style="margin-top:20px; overflow-y:auto; height:69%"></div>
```

Comment:
Can we add `box-shadow: none;` to the style? The shadow is a bit odd in the middle of the left panel.

Reply:
I do see how it could be useful to have some line to indicate the bottom of the scroll box when we add enough results for the results to go off the bottom of the scroll bar, but maybe there's a way to auto-scroll to where the latest result was added? Outside the scope of this pull request, of course.

Reply:
Yes I can absolutely do that! I personally like the shadow as a separation of divs, but it definitely looks a bit wonky especially when the modelling mode left side panel doesn't have the same thing, so I'll change it for this request! I'm just waiting for Esters comments to make final changes and push a commit. 

I think in a future world we have to figure out how to handle the watermark consistently across modes - you hit on a bunch of much larger issues that I also noticed but couldn't quite figure out how to fix at least without some bigger changes. This is where you come in Kathleen! Tbh if you as a css whiz have anything you want to add to this pull request you can def do it directly on the branch, but obviously no pressure at all!